vmdb/app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -145,8 +147,6 @@ def get_subnets
     end
 
     def get_volumes
-      # TODO: support volumes through :nova as well?
-      return unless @volume_service_name == :cinder
       process_collection(volumes, :cloud_volumes) { |volume| parse_volume(volume) }
```

Comment:
Not sure I understand how this would work if `volumes` can return nil (when service is cinder) that is passed into `process_collection` that calls `.each` on it.